cmake/BDWGC.cmake

```diff
@@ -37,12 +37,12 @@ macro(p4c_obtain_bdwgc)
     fetchcontent_declare(
       bdwgc
       GIT_REPOSITORY https://github.com/ivmai/bdwgc.git
-      GIT_TAG e340b2e869e02718de9c9d7fa440ef4b35785388 # 8.2.6
+      GIT_TAG 7f1503dbfe254e77678666a0e09b735add064b57 # 8.2.6, 2024-08-03
       GIT_PROGRESS TRUE
-      PATCH_COMMAND
-        git apply ${P4C_SOURCE_DIR}/cmake/bdwgc.patch || git apply
-        ${P4C_SOURCE_DIR}/cmake/bdwgc.patch -R --check && echo
-        "Patch does not apply because the patch was already applied."
+      # PATCH_COMMAND
+      #   git apply ${P4C_SOURCE_DIR}/cmake/bdwgc.patch || git apply
+      #   ${P4C_SOURCE_DIR}/cmake/bdwgc.patch -R --check && echo
+      #   "Patch does not apply because the patch was already applied."
```

Comment:
Since you have deleted the patch, I suggest you delete this too.

Reply:
Figured to keep this incantation around in case we need it again, but there are other places where it can be copied from.